source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -23,6 +24,8 @@ Breaking Changes
 
 - Trajectory constraint methods are now ``const`` in C++. Teams defining their own custom constraints should mark the ``MaxVelocity()`` and ``MinMaxAcceleration()`` methods as ``const``.
 
+- The ``Field2d`` class was moved from the simulation package (``edu.wpi.first.wpilibj.simulation`` / ``frc/simulation/``) to the SmartDashboard package (``edu.wpi.first.wpilibj.smartdashboard`` / ``frc/SmartDashboard/``). This allows teams to send their robot position over NetworkTables to be viewed in Glass.
```

Comment:
This should tell them to put it to smartdashboard in order to see it in the simgui.

Reply:
I think it's still not clear that you have to put it to SD in order to have the old behavior of it showing in the simgui